<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Windows</class>
 <widget class="QMainWindow" name="Windows">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>664</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Room Segmentation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/MainWidget.qrc">
    <normaloff>:/MainWidget/ico/MainWidget.ico</normaloff>:/MainWidget/ico/MainWidget.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="fileOpenFile"/>
    <addaction name="fileSave"/>
    <addaction name="separator"/>
    <addaction name="fileCloseAll"/>
    <addaction name="fileQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>显示</string>
    </property>
    <addaction name="console"/>
    <addaction name="menu"/>
    <addaction name="languages"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>插件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>菜单栏</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolBarOpenFile"/>
   <addaction name="toolBarSave"/>
   <addaction name="toolBarSeg"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetTree">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>DB树</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="autoExpandDelay">
        <number>-1</number>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetProperties">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>属性</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableWidget" name="tableWidgetProperties"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConsole">
   <property name="windowTitle">
    <string>控制台</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidgetConsole"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileOpenFile">
   <property name="icon">
    <iconset resource="../res/MainWidget.qrc">
     <normaloff>:/MainWidget/ico/folder.png</normaloff>:/MainWidget/ico/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="toolBarOpenFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../res/MainWidget.qrc">
     <normaloff>:/MainWidget/ico/folder.png</normaloff>:/MainWidget/ico/folder.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="fileQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="fileCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="toolBarSave">
   <property name="icon">
    <iconset resource="../res/MainWidget.qrc">
     <normaloff>:/MainWidget/ico/Save.png</normaloff>:/MainWidget/ico/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="fileSave">
   <property name="icon">
    <iconset resource="../res/MainWidget.qrc">
     <normaloff>:/MainWidget/ico/Save.png</normaloff>:/MainWidget/ico/Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="console">
   <property name="text">
    <string>控制台</string>
   </property>
  </action>
  <action name="menu">
   <property name="text">
    <string>菜单栏</string>
   </property>
  </action>
  <action name="languages">
   <property name="text">
    <string>语言</string>
   </property>
  </action>
  <action name="toolBarSeg">
   <property name="icon">
    <iconset resource="../res/MainWidget.qrc">
     <normaloff>:/MainWidget/ico/flash.png</normaloff>:/MainWidget/ico/flash.png</iconset>
   </property>
   <property name="text">
    <string>分割</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/MainWidget.qrc"/>
 </resources>
 <connections/>
</ui>
